### ISSUE TITLE
Support crafting

### ISSUE THREAD

**Issue by peacefulotter** [PrismarineJS/flying-squid#357]
The server crashes when I try to open a crafting table
(here is the error : https://hastebin.com/majonelaqo.coffeescript)

Thanks in advance :)

**Comment by rom1504** [comment]
there is no support for crafting yet

**Comment by Deudly** [comment]
Any of idea of what would be needed for this? Or any useful links/info of how does Minecraft handle this?

**Comment by rom1504** [comment]
yes, the idea is to implement the complementary of https://github.com/PrismarineJS/mineflayer/blob/master/lib/plugins/craft.js
